src/ui/public/stringify/types/_numeral.js

```diff
@@ -4,46 +4,47 @@ import { FieldFormat } from 'ui/index_patterns/_field_format/field_format';
 
 const numeralInst = numeral();
 
-_.class(Numeral).inherits(FieldFormat);
-export function Numeral(params) {
-  Numeral.Super.call(this, params);
-}
-
-Numeral.prototype._convert = function (val) {
-  if (val === -Infinity) return '-∞';
-  if (val === +Infinity) return '+∞';
-  if (typeof val !== 'number') {
-    val = parseFloat(val);
+export class Numeral extends FieldFormat {
+  constructor(params) {
+    super(params);
   }
 
-  if (isNaN(val)) return '';
+  _convert(val) {
+    if (val === -Infinity) return '-∞';
+    if (val === +Infinity) return '+∞';
+    if (typeof val !== 'number') {
+      val = parseFloat(val);
+    }
 
-  return numeralInst.set(val).format(this.param('pattern'));
-};
+    if (isNaN(val)) return '';
 
+    return numeralInst.set(val).format(this.param('pattern'));
+  }
+}
 
 Numeral.factory = function (opts) {
```

Comment:
You can use `static factory` in the class body definition, no need to separate this out.